cardano-cli/src/Cardano/CLI/EraBased/Query/Option.hs

```diff
@@ -215,50 +215,23 @@ pQueryCmds era envCli =
           , "obtained from the CARDANO_NODE_SOCKET_PATH environment variable."
           ]
     )
-    [ Just $
-        Opt.hsubparser $
-          commandWithMetavar "protocol-parameters" $
-            Opt.info (pQueryProtocolParametersCmd envCli) $
-              Opt.progDesc "Get the node's current protocol parameters"
-    , Just $
-        Opt.hsubparser $
-          commandWithMetavar "tip" $
-            Opt.info (pQueryTipCmd era envCli) $
-              Opt.progDesc "Get the node's current tip (slot no, hash, block no)"
-    , Just $
-        Opt.hsubparser $
-          commandWithMetavar "stake-pools" $
-            Opt.info (pQueryStakePoolsCmd era envCli) $
-              Opt.progDesc "Get the node's current set of stake pool ids"
-    , Just $
-        Opt.hsubparser $
-          commandWithMetavar "stake-distribution" $
-            Opt.info (pQueryStakeDistributionCmd era envCli) $
-              Opt.progDesc "Get the node's current aggregated stake distribution"
-    , Just $
-        Opt.hsubparser $
-          commandWithMetavar "stake-address-info" $
-            Opt.info (pQueryStakeAddressInfoCmd era envCli) $
-              Opt.progDesc $
-                mconcat
-                  [ "Get the current delegations and reward accounts filtered by stake address."
-                  ]
+    [ pQueryGetCommitteeStateCmd era envCli
+    , pQueryGetConstitutionCmd era envCli
+    , pQueryDRepStateCmd era envCli
+    , pQueryDRepStakeDistributionCmd era envCli
+    , pQueryEraHistoryCmd era envCli
+    , pQueryFuturePParamsCmd era envCli
+    , pQueryGetGovStateCmd era envCli
     , Just $
         Opt.hsubparser $
-          commandWithMetavar "utxo" $
-            Opt.info (pQueryUTxOCmd era envCli) $
-              Opt.progDesc $
-                mconcat
-                  [ "Get a portion of the current UTxO: by tx in, by address or the whole."
-                  ]
+          commandWithMetavar "kes-period-info" $
+            Opt.info (pKesPeriodInfoCmd era envCli) $
+              Opt.progDesc "Get information about the current KES period and your node's operational certificate."
```

Comment:
I discovered that if you use `.` instead of `$` except for the last one, `fourmolu` does something different:

```
    , Just
        . Opt.hsubparser
        . commandWithMetavar "kes-period-info"
        . Opt.info (pKesPeriodInfoCmd era envCli)
        $ Opt.progDesc "Get information about the current KES period and your node's operational certificate."
```

This seems to be to be tidier, and less indenting means there is more horizontal space for the description.

Thoughts?

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/commit/641c18f893f76061e2940ebb0f067221c5220e21